zipkin-ui/js/main.js

```diff
@@ -21,6 +22,7 @@ loadConfig().then(config => {
 
   crossroads.addRoute(contextRoot, () => initializeDefault(config));
   crossroads.addRoute(`${contextRoot}traces/{id}`, traceId => initializeTrace(traceId, config));
+  crossroads.addRoute('zipkin/traceViewer', () => initializeTraceViewer(config));
```

Comment:
shouldn't this be using the contextRoot variable? (if not, those proxy-mounting zipkin will be broken I think.. look at the README for more)

Reply:
Woops!  My bad.  Will fix.